### ISSUE TITLE
'state' is not defined  no-undef

### ISSUE THREAD

**Issue by robbporto** [babel/babel-eslint#682]
Hi, everyone.

I'm having a false positive when trying to declare `state` in a class component using react.

For example:

```
class ClassTeste extends Component {
  state = { loading: true } // ERROR HERE: 'state' is not defined  no-undef

  componentDidMount() {
  }

  componentDidUpdate(prevProps) {
  }
  
  render() {
    return <div>Teste</div>
  }
}
```

**Comment by rohan-deshpande** [comment]
Related to https://github.com/babel/babel-eslint/issues/529

**Comment by kaicataldo** [comment]
Thank you for the issue. Now that [`@babel/eslint-parser` has been released](https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint), we are making this repository read-only. If this is a change you would still like to advocate for, please reopen this in the [babel/babel monorepo](https://github.com/babel/babel).
